src/renderRepoCard.js

```diff
@@ -49,6 +50,14 @@ const renderRepoCard = (repo, options = {}) => {
   const totalStars = kFormatter(stargazers.totalCount);
   const totalForks = kFormatter(forkCount);
 
+  const template = `
```

Comment:
Is this the `template` badge? maybe rename it to `templateBadge` to be consistent with `archiveBadge`

Reply:
Yes, as like archive.